### ISSUE TITLE
Visual issue with the inline template picker and Large Text enabled (Accessibility)

### ISSUE THREAD

**Issue by maxme** [wordpress-mobile/gutenberg-mobile#2092]
**Describe the bug**
On a , with "Accessibility -> Large text enabled", the template picker is shown on top of the first paragraph:
![Screenshot_1585743676](https://user-images.githubusercontent.com/40213/78136702-5f9d2400-7424-11ea-847c-302c7806263f.png)

**To Reproduce**
1. Go to settings,  accessibility, enable "Large text".
1. Create a new page.

**Smartphone (please complete the following information):**
 - Nexus S, API 21
 - Android
 - wpandroid 14.5

**Comment by pinarol** [comment]
The transparency of those buttons were removed with [this PR](https://github.com/WordPress/gutenberg/pull/20959), so the overlapping should be fixed in the new release. This can be tested with parent app PRs here: https://github.com/wordpress-mobile/gutenberg-mobile/pull/2087

**Comment by jd-alexander** [comment]
I can confirm that this issue was resolved. 

Tested with Nexus S, Android 11, WP Android 15.7


![Screenshot_1600113253](https://user-images.githubusercontent.com/1509205/93131793-3c94c280-f69a-11ea-934a-de797fb79602.png)
